packages/components/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @baseapp-frontend/components
 
+## 0.0.59
+
+### Patch Changes
+
+- add filter by date
```

Comment:
please update versions
also, could you be more specific on the changes, like what module you are working on and a bit more details on the changes?